x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/external/http/sender/BaseRequestManager.java

```diff
@@ -37,4 +40,9 @@ public String inferenceEntityId() {
     public Object rateLimitGrouping() {
         return rateLimitGroup;
     }
+
+    @Override
+    public RateLimitSettings rateLimitSettings() {
```

Comment:
Not used yet but will be in a follow up PR. This will give `RequestExecutorService` access to the settings to construct a `RateLimiter`.